### PR TITLE
docs(devtools): add macOS instructions

### PR DESCRIPTION
**Pull request by vegerot**
(docs) Add macOS instructions for enable-devtools

### CODE REVIEW

**Review by CharlieS1103 — APPROVED** [review]
Seems good to me

**Review by theRealPadster — CHANGES_REQUESTED** [review]
If you're specifying each OS, then you need to mention Linux as well. I'd recommend just saying "ctrl/cmd + shift + I"

**Comment by vegerot** [comment]
> If you're specifying each OS, then you need to mention Linux as well. I'd recommend just saying "ctrl/cmd + shift + I"

but it's not shift, it's option (alt)

`ctrl/cmd + shift/option + I` sounds weird xD

**Comment by theRealPadster** [comment]
> but it's not shift, it's option (alt)
> 
> `ctrl/cmd + shift/option + I` sounds weird xD

Ahh that's why I could never get the shortcut to work on macos 😂 then I'd just add a "/Linux" to your "(Windows)"

File: spicetify.go [hunk 1]
```diff
@@ -285,7 +285,7 @@ restore             Restore Spotify to original state.
 clear               Clear current backup files.
 
 enable-devtools     Enable Spotify's developer tools.
-                    Hit Ctrl + Shift + I in the client to start using.
+                    Hit Ctrl + Shift + I (Windows) or Cmd + Option + I (macOS) in the client to start using.
```
Comment (theRealPadster) [inline spicetify.go:288]:
```suggestion
                    Press Ctrl + Shift + I (Windows/Linux) or Cmd + Option + I (macOS) in the Spotify client to start using.
```

**Review by theRealPadster — CHANGES_REQUESTED** [review]
I think this addresses all the feedback.

**Comment by vegerot** [comment]
@theRealPadster I don't have permission to merge this PR

**Pull request merged**